swap_meet/vendor.py

```diff
@@ -1,2 +1,61 @@
+import copy
+
+# Wave 1 
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=[]):
+        self.inventory = copy.copy(inventory)
+
+    def add(self, item): 
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        if item not in self.inventory: 
+            return False
+        else:
+            self.inventory.remove(item)
+            return item
+
+    # Wave 2
+    def get_by_category(self, category):
+        items = [item for item in self.inventory if item.category == category]
+        return items
```

Comment:
Nice use of a comprehension! One small thing: because we immediately return the variable, we could just say

```
return [item for item in self.inventory if item.category == category]
```